ui/scss/init/_gui.scss

```diff
@@ -2061,28 +2061,6 @@ input:not(input:-webkit-autofill)::-webkit-contacts-auto-fill-button {
     margin-top: 10px;
   }
 
-  .banner_logo {
-    display: block;
-    float: left;
-    margin-top: 20px;
-    margin-left: 80px;
-    margin-right: 30px;
-    width: 150px;
-    height: 150px;
-    background-image: url('/public/img/cookie.svg');
-    background-size: contain;
-    background-repeat: no-repeat;
-
-    @media (max-width: $breakpoint-medium) {
-      margin-left: 30px;
-      margin-right: 10px;
-    }
-
-    @media (max-width: $breakpoint-small) {
-      display: none;
-    }
-  }
-
```

Comment:
The comment says `Address webpack flagged issues`. 
- What is the issue, and where is the replacement?  This is still needed.